app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users, class_name: 'User'
+  belongs_to :posts
+
```

Comment:
### [OPTIONAL]
**Hint** : Just like we have in posts, we can do the same here `after_save :update_comments_counter`